apps/client/src/app/services/import-activities.service.ts

```diff
@@ -28,6 +28,7 @@ export class ImportActivitiesService {
     'unitprice',
     'value'
   ];
+  private static COMMENT_KEYS = ["comment", "note"]
```

Comment:
These values are good 👍🏻 

Can you please move this up ⬆️  to keep the alphabetical order?

Reply:
sure